tests/render/test_renderer.py

```diff
@@ -9,36 +9,6 @@
 from great_expectations.render.renderer.renderer import Renderer
 
 
-@pytest.mark.unit
```

Comment:
How is this test related to the v1 renaming?

Reply:
`v1_render` became `render` and these tests no longer worked. I read them over and I don't think they provide enough value so I fully deleted them